frontend/src/tests/UserNameInput.spec.ts

```diff
@@ -299,92 +363,89 @@ describe('UserNameInput', () => {
         expect(cardEl.textContent).toContain('Last Name');
         expect(cardEl.textContent).toContain('Organization Name');
         // verify identity card user info
-        expect(wrapper.find('#userId').element.textContent).toContain(USERID);
+        expect(wrapper.find('#userId').element.textContent).toContain(USER_ID);
         expect(wrapper.find('#firstName').element.textContent).toContain(
-            FIRSTNAME
+            FIRST_NAME
         );
         expect(wrapper.find('#lastName').element.textContent).toContain(
-            LASTNAME
+            LAST_NAME
         );
         expect(wrapper.find('#organizationName').element.textContent).toContain(
-            BUSINESSLEGALNAME
+            BUSINESS_LEGAL_NAME
         );
     });
 
-    it('Should show the user identity card with permission when BCEID user is not allowed to search', async () => {
+    it('should BCeID user able to verify other BCeID users and display info on card', async () => {
```

Comment:
Hi Jota, I wonder if this case this is same as the one above it?

Reply:
they are very similar, one spy on '`bceidSearch`' the other on '`idirSearch`' I think it can be put both on the same test case to be more concise.